src/components/IncomingMessageList/ConversationPreviewModal.jsx

```diff
@@ -161,34 +179,18 @@ class ConversationPreviewModal extends Component {
   }
 }
 
-ConversationPreviewModal.propTypes = {
+_ConversationPreviewModal.propTypes = {
+  organizationId: PropTypes.string,
   conversation: PropTypes.object,
-  onRequestClose: PropTypes.func
+  onRequestClose: PropTypes.func,
+  mutations: PropTypes.object,
+  onForceRefresh: PropTypes.func
 };
 
 const mapMutationsToProps = () => ({
-  createOptOut: (optOut, campaignContactId) => ({
-    mutation: gql`
-      mutation createOptOut(
-        $optOut: OptOutInput!
-        $campaignContactId: String!
-      ) {
-        createOptOut(optOut: $optOut, campaignContactId: $campaignContactId) {
-          id
-          optOut {
-            id
-            createdAt
-          }
-        }
-      }
-    `,
-    variables: {
-      optOut,
-      campaignContactId
-    }
-  })
+  createOptOut: createOptOutMutation
 });
 
-export default loadData(wrapMutations(ConversationPreviewModal), {
+export default loadData(wrapMutations(_ConversationPreviewModal), {
```

Comment:
if this item loads data, it should be a container and in the src/containers directory.  Please move.